sdc/rabbit/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .async_consumer import AsyncConsumer  # noqa
```

Comment:
Done - though not sure I agree that absolute imports are always better than explicit relative imports (though they are explicitly better than implicit relative imports :wink:).